plugin/trino-vertica/src/main/java/io/trino/plugin/vertica/VerticaClient.java

```diff
@@ -118,6 +123,8 @@ public class VerticaClient
             .appendValueReduced(ChronoField.YEAR, 4, 7, 1000)
             .appendPattern("-MM-dd[ G]")
             .toFormatter();
+    private static final DateTimeFormatter FORMATTER_ZERO_SCALE = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+    private static final DateTimeFormatter FORMATTER_SIX_SCALE = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss.SSSSSS");
```

Comment:
Is it possible to use optional fraction like `OracleClient.TIMESTAMP_NANO_OPTIONAL_FORMATTER`?